package.json

```diff
@@ -60,12 +60,12 @@
     "stylelint-config-recommended": "5.0.0",
     "stylelint-config-styled-components": "0.1.1",
     "stylelint-processor-styled-components": "1.10.0",
-    "typescript": "4.6.3"
+    "typescript": "4.8.2"
   },
   "husky": {
     "hooks": {
       "prepare-commit-msg": "exec < /dev/tty && git cz --hook || true",
-      "pre-commit": "lint-staged && yarn tsc:blade"
+      "pre-commit": "lint-staged"
```

Comment:
Lets add tsc for both native and web in pre-commit since the editor will only run tsc for one platform?

Reply:
Done!